modules/iam-assumable-role/main.tf

```diff
@@ -9,7 +9,7 @@ locals {
 }
 
 data "aws_iam_policy_document" "assume_role" {
-  count = var.custom_role_trust_policy == "" && var.role_requires_mfa ? 0 : 1
+  count = var.create_custom_role_trust_policy && var.role_requires_mfa ? 0 : 1
```

Comment:
The condition should be reverted to be the same as before. Now, `create_custom_role_trust_policy` is `false`, by default. Please verify.

Reply:
I don't really understand your point here, `create_custom_role_trust_policy = false` and `custom_role_trust_policy == ""` are equivalent. right?
Although, I found an issue happening when  `create_custom_role_trust_policy` was false and custom_role_trust_policy was not empty. I introduced a fix here: 1d9c0e43283ee6e22f637e18ec7fb4236e99b31c

Reply:
Done 84a47eacda09cb35960310bfe6c634f1b770ddc2